helm/alfresco-infrastructure/templates/pv-aws-efs.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.persistence.efs.enabled }}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ template "alfresco-infrastructure.fullname" . }}
+provisioner: alfresco.com/pv
+---
+apiVersion: v1
+kind: PersistentVolume
```

Comment:
Did this conversation continue offline?

Reply:
Yes. So we were still creating a pv here since we needed to specify the nfs target for the efs.
The provisioner is not the default option in the cluster.